### ISSUE TITLE
[FEATURE] templatize node count for remote execution in tensorflow_experiment.py

### ISSUE THREAD

**Issue by jiata** [microsoft/DistributedDeepLearning#7]
### Description
templatize node count for remote execution in tensorflow_experiment.py so that it equals the maximum number of nodes set during the cookie cutter setup.

By default, `node_count=4`, but i set my max nodes to 3, so it failed. 

### Expected behavior with the suggested feature
will not run into
```
"Requested 4 nodes but BatchAI cluster only has 3 maximum nodes.
```

### Other Comments

**Comment by msalvaris** [comment]
Good catch, will adress

**Comment by msalvaris** [comment]
Fixed in #12
